coverage/src/main/java/gov/cms/ab2d/coverage/service/CoverageService.java

```diff
@@ -76,16 +75,16 @@
 
     /**
      * Can an EOB search be started based on whether a contract mapping is in progress
-     * @return true if search {@link CoveragePeriod#getStatus()} is not {@link JobStatus#IN_PROGRESS}
+     * @return true if search {@link CoveragePeriod#getStatus()} is not {@link gov.cms.ab2d.common.model.JobStatus#IN_PROGRESS}
      */
     boolean canEOBSearchBeStarted(int periodId);
 
     /**
-     * Get search status {@link JobStatus}
+     * Get search status {@link gov.cms.ab2d.common.model.JobStatus}
      * @param periodId {@link CoveragePeriod#getId()}
      * @return search status or null
      */
-    JobStatus getSearchStatus(int periodId);
+    gov.cms.ab2d.common.model.JobStatus getSearchStatus(int periodId);
```

Comment:
Is the full package path really necessary in the code?

Reply:
Yes, because its in a new module It needs the full path. If not then it will fail for javadoc. Importing to keep javadoc causes test check fail due to unused import.

Reply:
Thanks for the explanation.